src/lib/useInviter.js

```diff
@@ -193,6 +193,8 @@ const useInviter = () => {
         };
       }
 
+      console.log('INVITES', confirmedInvites);
+      // TODO: store these locally, and also handle the situation
```

Comment:
Has this TODO been addressed?

Reply:
yes, we don't actually use that anymore so it has been removed